tests/test_storage/test_storage.py

```diff
@@ -17,6 +18,24 @@
 TEST_PATH = Path(__file__).parent
 
 
+def helper_function(func):
```

Comment:
What is this helper function and what does it help with?

Reply:
For tests that should be skipped, I think we can use https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-test-functions but for internal functions I would hope something like python's private naming convention _ would skip it, so naming the function something like 
```def _generate_dataframe(...```